notebooks/slsim_single_lens_simulation.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>this image you might also want to plot in color (or perhaps only this one)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>